CMSgov/mint-app
# [NOREF] Add new RDS Root CA

## Changes and Description

- Same changes as https://github.com/CMSgov/easi-app/pull/2632
- Per [RDS documentation](https://docs.aws.amazon.com/AmazonRDS/latest/UserGuide/UsingWithRDS.SSL.html), we need to rotate our RDS Root CA that's used by our instances, as the current one expires in August of this year.
- This step just adds the new default Root CA to our trust store for the application, and will allow us to rotate the instance to use the new certificate without downtime (besides the RDS maintenance window)
- Once the RDS instances in all environments are using the new certificate, we can remove the old certificate at [config/tls/rds-ca-2019-root.crt](config/tls/rds-ca-2019-root.crt)

![image](https://github.com/CMSgov/easi-app/assets/15203744/b2721a45-d50a-42a2-ada2-baa6ddfa8e25)
![image](https://github.com/CMSgov/easi-app/assets/15203744/5bb5984d-f8cd-4d5c-92d8-919d34ee1902)

## How to test this change

You can use the `openssl` tool as described below to ensure that the correct certificate has been added. To validate,
- Run `openssl x509 -text -noout -in config/tls/rds-ca-rsa2048-g1-us-east-1.crt`
- Ensure that the Issuer includes the correct AWS Region, and is RSA2048
- Ensure the expiry is from 2021 to 2061

## PR Author Review Checklist



## PR Reviewer Guidelines
- It's best to pull the branch locally and test it, rather than just looking at the code online!
- Check that all code is adequately covered by tests - if it isn't feel free to suggest the addition of tests.
- Always make comments, even if it's minor, or if you don't understand why something was done.
